src/ui/theme/shapes.ts

```diff
@@ -0,0 +1,25 @@
+export interface Shape {
+  "input-border-radius": string;
+  "input-button-border-radius": string;
+  "modal-border-radius": string;
```

Comment:
Hmm is it possible to name all these using camelCase instead? So we avoid using strings directly... Feels better to me to avoid strings, even if the compiler would still be able to catch most issues.

Edit, After more reviewing, I understand this is to make them compatible with CSS... I wonder if we should have a conversion method from the camelCase interface to a css-compatible map later on...

Reply:
I'm wondering if these should be numbers instead of strings. Just so it's easier to customize from the dashboard in the future if needed.

Reply:
On the camelCase:
I personally don't like when I have to remember conventions where "some-thing" is actually changed by setting "someThing". These are style variables and css prefers `-` to camelCase.
I would keep it simple, the mapping is already complex as is 😅 

On the numbers:
I would avoid it since the sizes can actually be expressed in different values than `px`, as example `em` or `%`.
This keeps that here and makes it look a bit more like CSS